core/src/scene/sceneLoader.cpp

```diff
@@ -560,7 +560,7 @@ std::shared_ptr<Texture> SceneLoader::fetchTexture(const std::string& name, cons
                 auto ptr = (unsigned char*)(rawData.data());
                 size_t dataSize = rawData.size();
                 std::lock_guard<std::mutex> lock(m_textureMutex);
-				auto texture = scene->getTexture(name);
+                                auto texture = scene->getTexture(name);
```

Comment:
Nice! looking good - even though I'm more in favor of four point-placement for better icon-text association :)

@varun: of course there is now more jumping as anchor fallbacks are not coordinated between zoom-levels. definitely something to investigate in future